src/main/scala/ignition/core/jobs/CoreJobRunner.scala

```diff
@@ -71,32 +76,39 @@ object CoreJobRunner {
       val appName = s"${config.setupName}.${config.tag}"
 
 
-      val sparkConf = new SparkConf()
-      sparkConf.set("spark.executor.memory", config.executorMemory)
-
-      sparkConf.set("spark.eventLog.dir", "file:///media/tmp/spark-events")
+      val builder = SparkSession.builder
```

Comment:
`builder` is state full? I always use it as stateless fluent interface / builder pattern

Reply:
It's a mutable hashmap